shiro3/build.gradle

```diff
@@ -12,11 +12,11 @@ buildscript {
 }
 
 plugins {
-    id "io.spring.dependency-management" version "0.4.0.RELEASE"
-    id "com.jfrog.bintray" version "1.1"
+    id "io.spring.dependency-management" version "0.5.2.RELEASE"
+    id "com.jfrog.bintray" version "1.2"
 }
 
-version "3.0-SNAPSHOT"
+version "0.1-SNAPSHOT"
```

Comment:
this should be version 3.0 in line with the major grails version this belongs to. We can use minor versions for tracking changes in shiro version and other updates.